src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.TestResultService/TestResultService.cs

```diff
@@ -48,7 +48,7 @@ public List<CompetitionResult> CalculateResults(DateTime dateFrom, DateTime date
                 var bonus = 0;
                 foreach (var correctAnswerDate in correctAnswerDates)
                 {
-                    var wrongAnswers = wrongAnswersCounts.FirstOrDefault(w => w.TestStartDate == correctAnswerDate);
+                    var wrongAnswers = wrongAnswersCounts.FirstOrDefault(w => w.TestStartDate.Date == correctAnswerDate.Date);
```

Comment:
what if null?
should it be
w.TestStartDate?.Date == correctAnswerDate?.Date

Reply:
This cannot be null.